src/system/applications/combat/combat-tracker.ts

```diff
@@ -0,0 +1,151 @@
+import CosmereCombatant from './combatant';
+
+export default class CosmereCombatTracker extends CombatTracker {
+    //overrides default tracker template to implement slow/fast buckets and activation button.
```

Comment:
This might be nice to include as a doc comment on the whole class

Reply:
Moved the comment to the whole class, please let me know if you want something more detailed. I'm used to working on solo hobby projects, so commenting is definitely an area I need to work on